pgml-cms/docs/api/sql-extension/pgml.rank.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Rank documents against a piece of text using the specified ranking model.
+---
+
+# pgml.rank()
+
+The `pgml.rank` function is used to rank text documents against some text. This function is primarly used as the last step in a search system where the results returned from the initial search are reranked before being used.
+
+## API
+
+```postgresql
+pgml.rank(
+    transformer TEXT,  -- transformer name
+    query TEXT,        -- text to rank against
+    documents TEXT[],  -- documents to rank
+    kwargs JSON        -- optional arguments (see below)
+)
+```
+
+## Example
+
```

Comment:
Let's document the difference between these calls. Take a look at examples:
- https://postgresml.org/docs/api/sql-extension/pgml.transform/#examples
- https://postgresml.org/docs/api/sql-extension/pgml.embed#generate-embeddings-inside-a-table

Reply:
How does this look: https://github.com/postgresml/postgresml/pull/1514/commits/f9c5ff2e82ecd96ffef0daaa08c92bb57099032c